boot/environ/include/efi/ProcessorBind.h

```diff
@@ -197,6 +197,7 @@ typedef signed char         CHAR8;
 typedef signed char         INT8;
 #endif
 
+#ifdef WIN32
```

Comment:
Is this piece of code could even be compiled with ``WIN32`` defined? 🤔 This code shouldn't end up in Win32 application